src/Gui/SoFCCSysDragger.cpp

```diff
@@ -96,7 +96,7 @@ TDragger::TDragger()
 {
     SO_KIT_CONSTRUCTOR(TDragger);
 
-#if defined(Q_OS_MACOS)
+#if defined(Q_OS_MAC) || defined(Q_OS_FREEBSD)
```

Comment:
The `Q_OS_MAC` define is a deprecated synonym for `Q_OS_DARWIN`.

Reply:
I've copied it from the FreeBSD port. I guess the define should still be `Q_OS_MACOS` as this is not marked as deprecated.